pkg/db/tracing.go

```diff
@@ -67,9 +67,9 @@ func (d traceableDB) QueryContext(ctx context.Context, query string, args ...int
 	defer func() {
 		d.FinishSpan(span, err)
 	}()
-	span.SetTag("sql", query)
+	span.LogKV("sql", query)
 
-	logrus.WithTime(time.Now()).WithField("sql.method", "QueryContext").Debug(query)
+	logrus.WithTime(time.Now()).WithField("sql_method", "QueryContext").Debug(query)
```

Comment:
using snake case integrates _much_ better with loki, it will autodetect these fields as metadata, when using `sql.method` is it not parsed automatically